packages/server/src/auth.ts

```diff
@@ -0,0 +1,55 @@
+import request from 'request';
+
+const { GITHUB_CLIENT_ID, GITHUB_CLIENT_SECRET, GITHUB_REDIRECT_URL } = process.env;
+const GENERIC_ERROR_STRING = 'An unexpected login error has occurred.';
+
+export interface IGithubAccessTokenResponse {
+  access_token: string;
+  error: string;
+  error_description: string;
+}
+
+export function getAccessTokenOrErrorResponse({
+  code,
+  state,
+}: {
```

Comment:
nit: I think it's cleaner to just make an interface for it so you can export it later if you ever need